content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -27,25 +39,35 @@ Our current process is as follows:
 
 - **While an iteration is ongoing, we plan the next iteration**. This is a mostly asynchronous process.
 
-  - Engineers, designer, PM and EM **_propose_ issues to be worked on** the week _before_ the mid-iteration [Monday sync](#weekly-sync) by adding them to the next iteration milestone and setting "Status" to "Proposed". This is done considering our [higher-level goals and roadmap](../../../../../strategy-goals/strategy/code-graph/code-insights/index.md).
+  - Engineers, designer, PM and EM can **_propose_ issues to be worked on** _before_ the mid-iteration [Thursday iteration planning](#fortnightly-iteration-planning) by adding them to the next iteration milestone and setting "Status" to `Proposed`. This is done considering our [higher-level goals and roadmap](../../../../../strategy-goals/strategy/code-graph/code-insights/index.md).
+
+  - We plan our iterations to 80% of team capacity to account for support, life events, and time off, and we plan that 80% capacity such that ~80% of our capacity is planned by the PM as the decider, and ~20% of the capacity is explicitly left for engineers to prioritize, taking into account our [goals and roadmap](../../../../../strategy-goals/strategy/code-graph/code-insights/index.md), technical roadmap, estimates, workloads on individuals, and release dates (the 20th of every month).<br> The PM will choose issues among all proposed issues – engineers and designers should still propose issues, even if they are not the planning decider.
```

Comment:
> We plan our iterations to 80% of team capacity to account for support, life events, and time off

I think this sentence was only relevant with day estimates, now that we use point baselines those factors would already be factored in. Instead, we should document that we do a temp check in our retro and manually adjust baselines.